### ISSUE TITLE
Globals settings need to be reset during tests

### ISSUE THREAD

**Issue by peterramsing** [peterramsing/lost#292]
**Is this a feature request or a bug report?**
Bug

**What is the current behavior?**
When a global is set in one test it persists throughout all tests

**If it's a bug please provide the steps to reproduce it and maybe some code samples.**
In a test, set a global. This global will persist for the rest of tests.

``` js
  it('supports clearing fallback', function() {
    check(
      '@lost clearing left; \n' +
      'a { lost-column: 1/3; }',
      'a { width: calc(99.99% * 1/3 - (30px - 30px * 1/3)); }\n' +
      'a:nth-child(1n) { float: left; margin-right: 30px; clear: none; }\n' +
      'a:last-child { margin-right: 0; }\n' +
      'a:nth-child(3n) { margin-right: 0; float: right; }\n' +
      'a:nth-child(3n + 1) { clear: left; }'
    );
  });
```

**What is the behavior that you expect?**
That to global will reset for each test.

**What's the motivation or use-case behind changing the behavior?**
Having to write a follow-up to the tests to reset the global

**What version of LostGrid, browser and browser version are affected by this issue? Did this happen in previous versions?**
6.8.0

**Comment by peterramsing** [comment]
Also see #261

**Comment by kfrost89** [comment]
I am stilling having these issues.

I have just tested using the shorthand syntax as well.

http://codepen.io/kfrost89/pen/ezYLoE

**Comment by peterramsing** [comment]
Closed by #312
